Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
 
 
 Bugfixes:
+* CompilerStack: Fix ICE that could occur when using the standard JSON interface for nondeployable contracts with ``outputSelection`` set to ``ir`` or ``irOptimized``.
```

Comment:
This is (luckily) not what happened, is it? 0.8.28 works just fine with ``ir`` or ``irOptimized`` requested.

Reply:
Yeah, it was about the IR AST outputs. If it was IR itself, we would have noticed pretty quickly :)

Reply:
Argh, yeah, ast outputs of course..